apps/prairielearn/elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -501,7 +526,14 @@ def test(element_html: str, data: pl.ElementTestData) -> None:
         }
 
 
-def create_table_for_html_display(m, n, name, label, data, format_type):
+def create_table_for_html_display(
+    m: int,
+    n: int,
+    name: str,
+    label: str | None,
+    data: pl.QuestionData,
+    format_type: Literal["output-invalid", "output-feedback", "input"],
+):
```

Comment:
Return type missing.

Reply:
Fixed in [3cd2306](https://github.com/PrairieLearn/PrairieLearn/pull/11188/commits/3cd23066a1f97e5fc88538e0cdc1a4511419e5ea).